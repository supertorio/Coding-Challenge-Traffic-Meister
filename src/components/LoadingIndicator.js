import React from 'react';

const LoadingIndicator = () => {
    return (
        <div className="LoadingIndicator">Loading</div>
    );
};

export default LoadingIndicator;
