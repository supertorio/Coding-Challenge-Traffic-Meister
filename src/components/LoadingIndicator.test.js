import React from 'react';
import LoadingIndicator from './LoadingIndicator';
import renderer from 'react-test-renderer';

describe('<LoadingIndicator />', () => {

    it('should match the snapshot', () => {
        const component = renderer.create(
            <LoadingIndicator />,
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});
