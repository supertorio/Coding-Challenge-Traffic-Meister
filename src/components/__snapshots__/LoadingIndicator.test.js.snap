// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingIndicator /> should match the snapshot 1`] = `ShallowWrapper {}`;
